packages/voyager-subscriptions/src/VoyagerSubscriptionContextProvider.ts

```diff
@@ -0,0 +1,97 @@
+// import { AuditLogger } from '@aerogear/voyager-audit'
+import { SecurityService } from '@aerogear/voyager-keycloak'
+
+import { VoyagerSubscriptionServerOptions } from './api'
+
+type OnConnectFunction = (connectionParams: any, webSocket: any, context: any) => any
+
+/**
+ * VoyagerSubscriptionContextProvider's purpose is to provide an
+ * 'onConnect' function that allows developers to supply their own onConnect
+ * the way they would with plain Apollo. However this onConnect will be combined
+ * with our own onConnect which may do some things such as authentication/authorization
+ * The results from our onConnect and the developer's onConnect functions are combined together
+ * and are made available in the GraphQL context inside subscription resolvers
+ *
+ */
+export class VoyagerSubscriptionContextProvider {
+
+  public onConnectFunction: any
```

Comment:
Missing type

Reply:
I have changed this to `Function` because that's the type the `SubscriptionServer` uses.